Assets/Scripts/Control&Input/PlayerMovement.cs

```diff
@@ -181,12 +183,23 @@ public void SetPlayerInput(InputManager player)
         localGunHolderX = gunHolder.transform.localPosition.x;
         localGunHolderHeight = gunHolder.transform.localPosition.y;
         playerCamera = inputManager.PlayerCamera;
-        startingFov = StaticInfo.Singleton.CameraFov;
+        SetFOVFromSettings();
+        sensScale = SettingsInfo.Singleton.settings.sensScale;
 
         if (MatchController.Singleton)
             MatchController.Singleton.onRoundEnd += ResetZoom;
     }
 
+    public void SetFOVFromSettings()
+    {
+        if (playerCamera != null)
+        {
+            startingFov = SettingsInfo.Singleton.settings.playerFOV;
+            playerCamera.fieldOfView = SettingsInfo.Singleton.settings.playerFOV;
+            ZoomFov = SettingsInfo.Singleton.settings.zoomFOV;
```

Comment:
Note that this is overridden by the telescope extension. Should be refactored to be a factor, probably, so that you can do `ZoomFovFactor * theActualFovFromSettings`